### PR TITLE
Update document for external network use case

### LINKED ISSUE #796: Update document for external network use case
Document for external network use case needs to be updated for
- headless service without selector support for globalnet
- service without selector support for light house
- remove description of experimental feature

### PR DESCRIPTION
**Pull request by mkimuram**
Fixes: #796

Depends on https://github.com/submariner-io/lighthouse/issues/817

### CODE REVIEW

**Comment by mkimuram** [comment]
Note that https://github.com/submariner-io/lighthouse/issues/817 needs to be resolved, before it works as described (Added a dependency to the issue in the first comment).

File: src/content/getting-started/quickstart/external/_index.md [hunk 1]
```diff
@@ -1,18 +1,14 @@
 ---
 date: 2021-08-11T12:33:18+01:00
-title: "External Network (Experimental)"
+title: "External Network"
```
Comment (sridhargaddam) [inline src/content/getting-started/quickstart/external/_index.md]: Masaki was one of the main users for the external network use-case and has added multiple features in Submariner (including e2e tests) to fill the missing gaps. I'm fine with removing the "Experimental" tag, but would like to hear the views of others.
CC: @nyechiel @aswinsuryan @mkolesnik @skitt @tpantelis @vthapar

**Review by sridhargaddam — COMMENTED** [review]
Thanks for pushing the PR @mkimuram

File: src/content/getting-started/quickstart/external/_index.md (hunk 1 above)
Comment (dfarrell07) [reply src/content/getting-started/quickstart/external/_index.md]: I think we should likely keep the "Experimental" tag.

Comment (dfarrell07) [reply src/content/getting-started/quickstart/external/_index.md]: @skitt (or another admin) can push an update to not-remove the "Experimental" line, and then we can review this and maybe-merge (as the code change is merged).

Comment (skitt) [reply src/content/getting-started/quickstart/external/_index.md]: I’ve pushed the update.

**Review by tpantelis — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**